backend/api/services/logger_service.js

```diff
@@ -0,0 +1,27 @@
+const { format, loggers, transports } = require('winston')
+
+const defaultFormat = format.printf(({ level, message, label, timestamp }) => {
+  return `${timestamp} ${level.toUpperCase()}: ${message}`;
+});
+
+const ts = {
+  error: new transports.File({ filename: 'appdata/logs/error.log', level: 'error' }),
+  combined: new transports.File({ filename: 'appdata/logs/combined.log' }),
+  console: new transports.Console({level: !debugMode ? 'info' : 'debug', name: 'console'})
+};
+
+loggers.add('yt-dl-material', {
```

Comment:
Is this loggers.add statement equivalent to the createLogger logic from before? Just wanna make sure this part is tested (as in the logger is working A-OK)